iam/quickstartv2/quickstart.go

```diff
@@ -0,0 +1,183 @@
+
+	// Gets the project's policy and prints all members with the "Log Writer" role
+	policy := getPolicy(crmService, *projectID)
+	var binding *cloudresourcemanager.Binding = nil
+	bindings := policy.Bindings
+	for b := range bindings {
+		if bindings[b].Role == role {
+			binding = bindings[b]
+			break
+		}
+	}
+	fmt.Println("Role: ", binding.Role)
+	fmt.Print("Members: ")
+	for m := range binding.Members {
+		fmt.Print("[", binding.Members[m], "] ")
+	}
+
+	// Removes member from the "Log writer" role
+	removeMember(crmService, *projectID, *member, role)
+
+}
+
+// initializeService initializes a new Cloud Resource Manager service
+func initializeService() *cloudresourcemanager.Service {
+
+	ctx := context.Background()
+	crmService, err := cloudresourcemanager.NewService(ctx)
+	if err != nil {
+		log.Fatalf("cloudresourcemanager.NewService: %v", err)
+	}
+	return crmService
+
+}
+
+// addBinding adds the member to the project's IAM policy
+func addBinding(crmService *cloudresourcemanager.Service, projectID, member, role string) {
+
+	policy := getPolicy(crmService, projectID)
+
+	// Finds the role binding in the policy, if it exists
+	bindings := policy.Bindings
+	var binding *cloudresourcemanager.Binding = nil
+	for b := range bindings {
+		if bindings[b].Role == role {
+			binding = bindings[b]
+			break
+		}
+	}
+
+	if binding != nil {
+		// If the binding exists, adds the member to the binding
+		binding.Members = append(binding.Members, member)
+	} else {
+		// If the binding does not exist, adds a new binding to the policy
+		binding = new(cloudresourcemanager.Binding)
+		binding.Role = role
+		binding.Members = []string{member}
+		policy.Bindings = append(policy.Bindings, binding)
+	}
+
+	setPolicy(crmService, projectID, policy)
+
+}
+
+// removeMember removes the member from the project's IAM policy
+func removeMember(crmService *cloudresourcemanager.Service, projectID, member, role string) {
+
+	policy := getPolicy(crmService, projectID)
+
+	// Finds the binding in the policy
+	bindings := policy.Bindings
+	var binding *cloudresourcemanager.Binding = nil
+	var bindingIndex int
+	for b := range bindings {
+		if bindings[b].Role == role {
+			binding = bindings[b]
+			bindingIndex = b
+			break
+		}
+	}
+
+	// Order doesn't matter for bindings or members, so to remove, move the last item
+	// into the removed spot and shrink the slice.
+	if len(binding.Members) == 1 {
+		// If the member is the only member in the binding, removes the binding
+		last := len(bindings) - 1
+		bindings[bindingIndex] = bindings[last]
+		bindings[last] = nil
+		policy.Bindings = bindings[:last]
+	} else {
+		// If there is more than one member in the binding, removes the member
+		var memberIndex int
+		for i, mm := range binding.Members {
+			if mm == member {
+				memberIndex = i
+			}
+		}
+		last := len(bindings[bindingIndex].Members) - 1
+		binding.Members[memberIndex] = binding.Members[last]
+		binding.Members[last] = ""
```

Comment:
Why zero this out?

Reply:
You're right, there's no reason to. Removed.